Makefile

```
@@ -126,20 +126,24 @@ ZAPD       := tools/ZAPD/ZAPD.out
 FADO       := tools/fado/fado.elf
 
 ifeq ($(COMPILER),gcc)
-  OPTFLAGS := -Os -ffast-math -fno-unsafe-math-optimizations
+	ifeq ($(DEBUG_BUILD),1)
+		OPTFLAGS := -Og -g -g3 -ffast-math -fno-unsafe-math-optimizations
```

`-g -g3` is the same as `-g3`
`-ggdb` also exists and is supposedly better when using gdb

```suggestion
		OPTFLAGS := -Og -ggdb3 -ffast-math -fno-unsafe-math-optimizations
```
I recommend reading `man gcc` about -g options